openedx/core/djangoapps/content/block_structure/tests/test_signals.py

```diff
@@ -22,25 +21,21 @@ def setUp(self):
         self.course = CourseFactory.create()
         self.course_usage_key = self.store.make_course_usage_key(self.course.id)
 
-    def test_course_publish(self):
```

Comment:
Discussed in person, testing was simplified for minimal code that exists in new location.